frontend/apps/reader/modules/readerstatus.lua

```diff
@@ -222,7 +223,8 @@ function ReaderStatus:deleteFile(file, text_end_book)
             if FileManager.instance then
                 FileManager.instance.file_chooser:refreshPath()
             else
-                FileManager:showFiles()
+                local path, filename = util.splitFilePathName(file)
```

Comment:
```suggestion
                local path = util.splitFilePathName(file)
```
Otherwise LuaCheck will complain.